### PR TITLE
Add adafruit-blinka to requirements.txt

### PR DESCRIPTION
**Pull request by tekktrik**
Add adafruit-blinka to requirements.txt

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@
 # SPDX-License-Identifier: Unlicense
 
 adafruit-circuitpython-display-text
+adafruit-blinka
```
Comment (kattni) [inline requirements.txt]: I'm not sure it really matters, but for the sake of consistency, please capitalise `Adafruit-Blinka`.
```suggestion
Adafruit-Blinka
```

**Review by kattni — CHANGES_REQUESTED** [review]
Thank you for submitting this! Please see the suggestion below.

File: requirements.txt (hunk 1 above)
Comment (tekktrik) [reply requirements.txt]: Roger roger, updated!

**Review by kattni — APPROVED** [review]
Thanks for adding this!

**Comment by tekktrik** [comment]
My pleasure!

**Pull request merged**